my_class.cpp

```diff
@@ -0,0 +1,165 @@
+#include "my_class.h"
+#include <iostream>
+
+using std::cin;
+using std::cout;
+using std::endl;
+using std::string;
```

Comment:
At least in the university explained us that using namespace includes all identifiers, while std::cin, std::cout, std::endl includes only these three. It is just an easier way to use, instead of adding std:: in front of each of them.